Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1191,24 +1192,50 @@ class BrowserViewController: UIViewController {
         tab.desktopSite = webView.customUserAgent?.isEmpty == false
     }
     
-    // MARK: DuckDuckGo Callout
+    // MARK: - Browser PIN Callout
+    
+    private var isBrowserLockEnabled: Bool {
+        return KeychainWrapper.sharedAppContainerKeychain.authenticationInfo() != nil
+    }
+    
+    func presentBrowserLockCallout() {
+        if isBrowserLockEnabled || Preferences.Popups.browserLock.value { return }
+        
+        let popup = AlertPopupView(image: UIImage(named: "browser_lock_popup"), title: Strings.Browser_lock_callout_title, message: Strings.Browser_lock_callout_message)
```

Comment:
I would set UIImage with `imageLiteral` constructor here and break this line, rest looks good for me. I also did few tests on simulator.